### PR TITLE
Agt2hc/feat/44/make documents and tutorial available in workspace

### PR DESCRIPTION
**Pull request by tranhoangnguyen193**
This pull request:
- Implemented issues #43 , add Project selection for Windows installer laucher
- Filter the tutorial subfolders (issue #37 ) 
- Add only Roboframework-reference.pdf file in book folder to RobotTest documents ( issue #44 )

### CODE REVIEW

File: config/projects/G3g/RobotTest_config.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+	"Name": "G3G Configuration"
```
Comment (test-fullautomation) [inline config/projects/G3g/RobotTest_config.json]: Hi Nguyen,
G3g is Bosch internal.
Therefore the best is only to have a generic configuration for 0.7.0 version.
For 0.8.0 we look how to improve.
Thank you,
 Thomas

File: scripts/RobotFrameworkSetup.iss [hunk 2]
```diff
@@ -34,6 +34,7 @@
 #define MyAppVersion "RobotFramework AIO " + RobotFrameworkVersion + " (Installer " + SETUPVersion + ")"
 #define MyAppFileName "RobotFramework_AIO_setup_" + RobotFrameworkVersion
 #define MyAppPublisher "Robert Bosch GmbH"
+#include '..\Output\Include\install_projects.iss';
```
Comment (test-fullautomation) [inline scripts/RobotFrameworkSetup.iss:37]: Good idea to load this from outside.
Please consider, that a manual build need to be possible to...

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Nguyen,

please read my finding.

Thank you,
 Thomas

File: config/projects/G3g/RobotTest_config.json (hunk 1 above)
Comment (tranhoangnguyen193) [reply config/projects/G3g/RobotTest_config.json]: Hi Thomas.
I will remove it in next commit.

File: scripts/RobotFrameworkSetup.iss (hunk 2 above)
Comment (tranhoangnguyen193) [reply scripts/RobotFrameworkSetup.iss:37]: Hi Thomas,

I tested it manual in my PC. The manual build was not effect by this setup.
When user run ./build the precompiler.bat (which is already called in ./build) will read the json config and generate '..\Output\Include\install_projects.iss'.
There are no more action for manual build now. For gitlab build, we should input the path of internal json config to ./build . I will push these commits into gitlab.

Comment (test-fullautomation) [reply scripts/RobotFrameworkSetup.iss:37]: O.K. very good!

File: config/projects/G3g/RobotTest_config.json (hunk 1 above)
Comment (tranhoangnguyen193) [reply config/projects/G3g/RobotTest_config.json]: Hi Thomas,
The internal project configs is removed.

**Review by test-fullautomation — APPROVED** [review]

**Pull request merged**